nettrine/app/config/common.neon

```diff
@@ -32,13 +34,33 @@ extensions:
 
 nettrine.dbal:
 	connection:
-		driver: 'pdo_sqlite'
-		url: sqlite:///%appDir%/../data/database.sqlite
+		driver: mysqli
```

Comment:
Proc nestaci sqlite?

Reply:
protože sqlite mi pořád házela nekonzitestnost DB s entitami a do migrací přívalo nesmysly do nekonečna, tak jsem to vrátil k vému mysql